### ISSUE TITLE
No Response from GDG support.

### ISSUE THREAD

**Issue by Mowahed** [gdg-wisdom/wisdom-2017#8]
I applied to open a GDG chapter in Kabul, Afghanistan, I received an email that they would let me know within a week or two whether the application is accepted or not. I had applied on 12th December,2017 it has been more than a month and despite a couple of reminders i haven't received any response. 
Searching the net i found that another fellow applicant had the same problem and got a response here. 
Should I expect a response? I would be grateful if you helped me out with this, Thanks

**Comment by friedger** [comment]
Maybe try twitter or so to contact them directly.



[image: --]
Friedger Müffke
[image: http://]friedger.de
<https://friedger.de>


2018-01-27 21:39 GMT+01:00 Jawid Mowahed <notifications@github.com>:

**Comment by Mistic92** [comment]
Did you get response?

**Comment by Mowahed** [comment]
Yes, We are now officialy GDG Kabul and we are organizing our first event which is on 20th April 2018.

**Comment by inesby** [comment]
Hello, we applied to start a GDG chapter in Manouba, Tunisia. I applied from 16 august and we are 30 september now. Please do anyone one knows how to contact them to have confirmation ?
